config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigtableinstances.bigtable.cnrm.cloud.google.com.yaml

```diff
@@ -70,12 +73,15 @@ spec:
                         cpuTarget:
                           description: The target CPU utilization for autoscaling.
                             Value must be between 10 and 80.
+                          format: int64
```

Comment:
Is this being explicit or does this represent a change?

Reply:
Ugh .. good call...

/hold

Reply:
Actually, it's a change, but I think it's a good change.  We should not be using `integer` in CRDs, we should be using int32 or int64.  Here it's an int64 according to the proto.

Reply:
(And we'll validate that it doesn't need to be quoted in our tests)